Sources/Logging/Logging.swift

```diff
@@ -264,6 +269,7 @@ extension Logger {
     }
 
     #else
```

Comment:
Why the inlinables here? We use this exclusively as existentials, right?

Reply:
All other methods (info, warning, error, critical) had them so it seemed to me those were omitted accidentally? maybe there's some history to this though?